### PR TITLE
[Feature] Linear onboarding flow

### PR DESCRIPTION
**Pull request by David-Henner**
## What's new in this PR?

This PR updates the authentication flow for Milestone 2 of the Linear Onboarding.
It updates the current company login flow to use email as an alternative to the SSO code, in order to connect to a custom backend.

### Dependencies
- [ ] https://github.com/wireapp/wire-ios-sync-engine/pull/1160

### CODE REVIEW

File: Wire-iOS/Resources/Base.lproj/Localizable.strings [hunk 1]
```diff
@@ -1372,22 +1374,26 @@
 "registration.no_history.restore_backup_failed.wrong_version.message" = "This backup was created by a newer or outdated version of Wire and cannot be restored here.";
 
 // SSO – Company Login
-"login.sso.alert.title" = "Enterprise Log In";
-"login.sso.alert.message" = "Enter your email or SSO access code here.";
+"login.sso.alert.title" = "Enterprise Login";
+"login.sso.alert.message.ssoAndEmail" = "Please enter your email or SSO code:";
```
Comment (typfel) [inline Wire-iOS/Resources/Base.lproj/Localizable.strings]: We use snake case in in the strings files.

File: Cartfile.resolved [hunk 2]
```diff
@@ -16,13 +16,13 @@ github "wireapp/swift-protobuf" "1.5.0"
...
-github "wireapp/wire-ios-data-model" "207.0.0"
+github "wireapp/wire-ios-data-model" "207.1.0"
 github "wireapp/wire-ios-images" "29.1.0"
 github "wireapp/wire-ios-link-preview" "25.1.0"
 github "wireapp/wire-ios-protos" "21.0.0"
-github "wireapp/wire-ios-request-strategy" "183.0.0"
+github "wireapp/wire-ios-request-strategy" "183.1.0"
 github "wireapp/wire-ios-share-engine" "168.0.0"
-github "wireapp/wire-ios-sync-engine" "296.0.0"
+github "wireapp/wire-ios-sync-engine" "fff4a5187e1194c5087f0bd8fbcd84cdd077fd6e"
```
Comment (billypchan) [inline Cartfile.resolved]:
```suggestion
github "wireapp/wire-ios-sync-engine" "feature/parse-company-login-input"
```

**Comment by billypchan** [comment]
@David-Henner You should mention the required PR of sync engine in the description so that we can track this PR is ready for merge or not.

File: Cartfile.resolved (hunk 2 above)
Comment (David-Henner) [reply Cartfile.resolved]: Thanks

**Pull request merged**